app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, dependent: :destroy
+  has_many :comments
+  has_many :likes
+
+  def recent_posts
+    Post.where(author_id: id).order(created_at: :desc).limit(3)
+  end
```

Comment:
- [ ] OPTIONAL: You can just write 

```ruby
def recent_post
  posts.order('created_at Desc').limit(3)
end
```

Rails is smart enough to find the posts that belong to this particular user, so you do not have to explicitly select posts with  `Post.where` 😉